packages/core.json

```diff
@@ -282,10 +224,7 @@
       }
     ],
     "new-parens": 2,
-    "newline-after-var": [
-      0,
-      "always"
-    ],
+    "newline-after-var": [0, "always"],
```

Comment:
not needed imo

Reply:
0 is off (see https://eslint.org/docs/user-guide/configuring#configuring-rules) so it will not throw an error or warning.